crates/tauri/src/manager/mod.rs

```diff
@@ -333,25 +333,9 @@ impl<R: Runtime> AppManager<R> {
     self.state.clone()
   }
 
-  /// Get the base path to serve data from.
-  ///
-  /// * In dev mode, this will be based on the `devUrl` configuration value.
-  /// * Otherwise, this will be based on the `frontendDist` configuration value.
-  #[cfg(not(dev))]
-  fn base_path(&self) -> Option<&Url> {
-    use crate::utils::config::FrontendDist;
-    match self.config.build.frontend_dist.as_ref() {
-      Some(FrontendDist::Url(url)) => Some(url),
-      _ => None,
-    }
-  }
-
-  #[cfg(dev)]
-  fn base_path(&self) -> Option<&Url> {
-    self.config.build.dev_url.as_ref()
-  }
-
-  pub(crate) fn protocol_url(&self, https: bool) -> Cow<'_, Url> {
+  /// The `tauri` custom protocol URL we use to serve the embedded assets,
+  /// returns `tauri://localhost` or its `wry` work around URL `http://tauri.localhost`/`https://tauri.localhost`
```

Comment:
```suggestion
  /// The `tauri` custom protocol URL we use to serve the embedded assets.
  /// Returns `tauri://localhost` or its `wry` workaround URL `http://tauri.localhost`/`https://tauri.localhost`
```
not sure about the wry workaround part at all tbh but whatever, better than before your PR.

Reply:
Somewhat talking about https://github.com/tauri-apps/wry/pull/1645